starter-code/MyLinkedList.java

```diff
@@ -13,14 +13,39 @@ public int getSize() {
 
 	public boolean remove(int index) {
 		//to-do
+		if (index > size) {
+			return false;
+		}
+		Node<T>node = head;
+		Node<T>afterNode;
+		for (int i = 0; i < index-1; i++) {
+			node = node.getNext();
+		}
+		afterNode = node.getNext();
+		node.setNext(afterNode);
```

Comment:
What if `index` is zero, i.e the caller wants to remove the head? I don't think your method will do that